static/css/main.css

```diff
@@ -39,6 +39,12 @@ ul {
   width: 40px;
 }
 
+.twitter {
```

Comment:
Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

*Origin: CSSLintBear, Section: `css`.*